sycl/doc/extensions/USMAddressSpaces/usm_address_spaces.asciidoc

```diff
@@ -28,14 +25,30 @@ Because the interfaces defined by this specification are not final and are subje
 == Version
 
 Built On: {docdate} +
-Revision: 1
+Revision: B
```

Comment:
I see that most of other extensions use numbers for revision.
So instead of fixing it here, I suggest we use numbers and fix the table at the end of the document.

Reply:
Fixed in https://github.com/intel/llvm/pull/4432/commits/d849467ac1826aaee7997d97850825c5182c6c0c